x-pack/plugins/aiops/public/components/change_point_detection/change_point_detection_page.tsx

```diff
@@ -169,17 +230,24 @@ export const ChangePointDetectionPage: FC = () => {
                       ) : null}
                     </EuiFlexGroup>
                   </EuiFlexItem>
+                </EuiFlexGroup>
+
+                <EuiHorizontalRule margin="xs" />
+
+                <EuiFlexGroup justifyContent={'spaceBetween'} alignItems={'center'}>
+                  {v.p_value !== undefined ? (
+                    <EuiFlexItem grow={false}>
+                      <EuiDescriptionList
+                        type="inline"
+                        listItems={[{ title: 'p-value', description: v.p_value.toPrecision(3) }]}
```

Comment:
In the log rates spikes page we flag `p-value` for translation.
https://github.com/elastic/kibana/blob/4a2cd71b99ea7c06b128cf3387d91fa146a2b7e3/x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx#L387

Reply:
Added the same translation string in 6e6be1c62f52994db055b5222fb755229aae2a26